### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jliaz**
* add a more descriptive/accurate description to described command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ cargo build
 
 ### Update Project Dependencies
 
-To build the project, use:
+To updat the project dependencies, use:
```
Comment (JustinReiter) [inline README.md]: You misspelled `update`

**Review by AmanPal-Singh — CHANGES_REQUESTED** [review]
Fix typo

**Review by JustinReiter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jliaz) [reply README.md]: Oops! Thanks for the catch :)

**Review by AmanPal-Singh — APPROVED** [review]

**Comment by BenjaminBLi** [comment]
squash your commits

**Comment by AmanPal-Singh** [comment]
image

**Comment by jliaz** [comment]
> squash your commits

Please approve my changes when you have the bandwidth :)

**Comment by JustinReiter** [comment]
Closing for inactivity

**Comment by AmanPal-Singh** [comment]
Can we reopen this PR? Issue is not resolved.

**Pull request closed without merging**